### DISCUSSION TITLE
Command Line Tools?

### DISCUSSION THREAD

**Discussion by jonathanware** [zwave-js/zwave-js-ui#2472] [Q&A]
I have a thermostat (CT100) that will send relative humidity (RH) over MQTT if I manually refresh values, but (and I think this is a bug of the thermostat) it doesn't send that info when the RH value changes. I was thinking I could set up a CRON job or something to run every 5-10 minutes to manually poll for that information, but I couldn't find any way of submitting that request other than through the web interface.

Anybody have any ideas of how I could do this?

**Comment by robertsLando** [comment]
Alternatives:

- Cron job that uses mosquitto pub to trigger a refresh of that value using mqtt apis
- node-red
- websocket (undocumented as it is just used on ui side right now)
